_posts/2018-04-28-evm-basics.md

```diff
@@ -0,0 +1,142 @@
+---
+layout:     post
+title:      Ethereum Virtual Machine in Elixir
+date:       2018-03-25
+summary:    EVM execution basics in Elixir
+categories: elixir
+---
+
+![ethereum](https://i.imgur.com/zaIytOk.jpg)
+
+### Introduction
+
+This week we, at Exthereum (Mana?) project, finally made all official EVM tests pass. I think it's time to write a post describing its inner magic with code examples from our project. So every crypto alchemist<a name="footnote1">1</a> interested in Ethereum can follow along.
```

Comment:
I think it should be Mana. Although, if you'd like to have it as Exthereum in your post I understand